src/segment_regquery_windows.go

```diff
@@ -0,0 +1,143 @@
+//go:build windows
+
+// TODO:
+//
+//  * reg value -> string formatting to own fn?
+//  * support for other reg types?
+//  * custom formatting for values?  (custom sprintf is too dangerous, what else?)
+//
+
+package main
+
+import (
+	"fmt"
+	"strings"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
```

Comment:
I only want to see code like this in `environment_windows.go`, it's not testable (at least not without Windows).

Reply:
done